zaza/openstack/utilities/generic.py

```diff
@@ -850,3 +850,22 @@ def systemctl(unit, service, command="restart"):
         unit.entity_id, cmd)
     assert int(result['Code']) == 0, (
         "{} of {} on {} failed".format(command, service, unit.entity_id))
+
+
+def get_mojo_cacert():
+    """Retrieve cacert from Mojo storage location.
+
+    :returns: Pathh to cacert
+    :rtype: str
+    :raises: zaza_exceptions.CACERTNotFound
```

Comment:
If you wanted to be super correct here: ":raises: :class:``zaza_exceptions.CACERTNotfound`"